.github/workflows/release-version.yml

```diff
@@ -0,0 +1,91 @@
+# This workflow automatically creates a release PR for the CI integration.
+
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: "Semver argument for the version bump to do on the CI integration"
+        default: "minor"
+        type: choice
+        options:
+          - "minor"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      # Do the changes
+      - uses: actions/checkout@v3
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+      - name: Create release branch
+        run: git checkout -b local-branch
+      - name: Set git user
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email noreply@github.com
+      - name: Install dependencies
+        run: yarn install
+      - name: Bump CI integration version
+        run: yarn version --${{ github.event.inputs.semver }}
+      - name: Update build files
+        run: yarn build && yarn package
+      - name: Amend version commit
+        id: amend-version-commit
+        run: |
+          VERSION_TAG=$(git tag --points-at HEAD)
+          git add --all
+          git commit --amend --no-edit
+          git tag --force $VERSION_TAG
+          echo "VERSION_TAG=$VERSION_TAG" >> $GITHUB_OUTPUT
+      - name: Push the branch (with tags)
+        run: git push --follow-tags -u origin local-branch:release/${{ steps.amend-version-commit.outputs.VERSION_TAG }}
```

Comment:
Github workflow expressions are documented here: https://docs.github.com/en/actions/learn-github-actions/expressions
It asks a bit of getting used to it, and we must be careful as to when is their evaluation. We could add a few bash comment within the commands to explain them.

Most notably because of the implicit dependency between the `$GITHUB_OUTPUT` and the `steps.<step-name>.outputs` fields.